deploy/helm/templates/swo-agent-statefulset.yaml

```diff
@@ -0,0 +1,39 @@
+{{- if .Values.swoagent.enabled }}
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ .Values.cluster.name | replace " " "-" }}-swo-agent
+  namespace: {{ .Release.Namespace }}
+  labels:
+{{ include "common.labels" . | indent 4 }}
+spec:
+  serviceName: {{ .Values.cluster.name | replace " " "-" }}-swo-agent
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ .Values.cluster.name | replace " " "-" }}-swo-agent
+  template:
+    metadata:
+      annotations:
+        checksum/values: {{ toJson .Values | sha256sum }}
+      labels:
+{{ include "common.template-labels" . | indent 8 }}
+        app: {{ .Values.cluster.name | replace " " "-" }}-swo-agent
+    spec:
+      securityContext: {}
+      containers:
+        - name: swo-agent 
+          image: {{ .Values.swoagent.image }}
+          env:
+            - name: UAMS_CLIENT_ID_OVERRIDE_SOURCE_NAME
+              value: {{ .Values.cluster.uid }}
+            - name: SWO_URL 
+              value: {{ trimPrefix "otel.collector" .Values.otel.endpoint }}
```

Comment:
Why are we modifying the `.Values.otel.endpoint` with the `trimPrefix` function?

Reply:
As we are suppling otel.collector.<swo_url> where swo url can be na-01.cloud.solarwinds.com or na-02.cloud.solarwinds.com or any other datacell. I am just reusing the same parameter, so we dont need to change wizard.

Reply:
In our wizard we prefill .Values.otel.endpoint to following value: `otel-collector.dc-01.cloud.solarwinds.com:443`, so it won't work anyway, you must cover also `otel-collector`

Reply:
`otel-collector` is deprecated and we should not use it anyway. but agree, we need to support both.